src/CMakeLists.txt

```diff
@@ -84,10 +100,17 @@ elseif(MINGW)
 
 endif()
 
-# Set the MSVC debug working directory, enforce conformance mode for osp-magnum
+# MSVC quality of life improvements for Visual Studio
 if(MSVC)
-  set_property(TARGET osp-magnum PROPERTY VS_DEBUGGER_WORKING_DIRECTORY ../../bin)
-  target_compile_options(osp-magnum PRIVATE $<$<CXX_COMPILER_ID:MSVC>:/permissive->)
+	# Set the MSVC debug working directory
+  	set_property(TARGET osp-magnum PROPERTY VS_DEBUGGER_WORKING_DIRECTORY ../../bin)
```

Comment:
imho it's unnecessary to make these items conditional on if(MSVC) None of them should cause problems outside of the MSVC block.

Reply:
I guess that's true, but if they have no effect outside of MSVC, why not make them conditional? Just for cleanliness' sake. I don't really know CMake so I was just playing it safe.

Reply:
MSVC can be used to compile for linux

and MSVC probably isn't the only IDE that would look at that flag. It might be the only one now, but others might look in the future.